docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1444,6 +1444,28 @@ rpc-ws-port="6174"
 Specifies Websockets JSON-RPC listening port (TCP).
 The default is 8546. Ports must be [exposed appropriately](../../HowTo/Find-and-Connect/Configuring-Ports.md).
 
+### target-gas-limit
+
+```bash tab="Syntax"
+--target-gas-limit=<INTEGER>
+```
+
+```bash tab="Command Line"
+--target-gas-limit=8000000
+```
+
+```bash tab="Environment Variable"
+BESU_TARGET_GAS_LIMIT=8000000
+```
+
+```bash tab="Configuration File"
+target-gas-limit="8000000"
+```
+
+Specifies the gas limit toward which Besu will gradually move on an existing network, if enough miners or validators are in agreement. Use `target-gas-limit` to change the block gas limit without creating a new network. The gas limit between blocks can change only 1/1024th, so the target tells the block creator how to set the gas limit in its block. The block gas limit can move up to 1/1024th of the gas limit between blocks. The block creator moves the block gas limit as far as it can in the direction of the target gas limit.
+    
+If a value for `target-gas-limit` is not specified, the block gas limit remains at the value specified in the [genesis file](../Config-Items.md#genesis-block-parameters).
```

Comment:
```suggestion
If a value for `target-gas-limit` is not specified, the block gas limit is not changed between blocks.
```
We respect the current network value regardless of what the genesis file says (assuming it follows the rules in its movement)